build.gradle

```diff
@@ -7,6 +8,17 @@ plugins {
 
 group 'io.github.cosmicsilence'
 
+gradlePlugin {
+    plugins {
+        scalafix {
+            id = 'io.github.cosmicsilence.scalafix'
+            implementationClass = 'io.github.cosmicsilence.scalafix.ScalafixPlugin'
+        }
+    }
+}
+
+version '0.0.1-SNAPSHOT'
```

Comment:
why do we need the changes above?

Reply:
Apologies, that was a merge conflict I didn't solve properly :)

Reply:
regarding the overlapping configuration of the plugin: I did some research on this and found this: https://docs.gradle.org/5.6.2/userguide/test_kit.html#sub:test-kit-automatic-classpath-injection . After playing around with the properties file and the snippet you added, it looks like having only the properties file is enough provided that you run tests using gradle as runner. If you attempt to run the test through Intellij then they will fail.

I also tested with only the config in build.gradle (deleted the properties file(. Looks like that is enough to get the plugin working. I tested it on my scala test project and it worked. I'd vote for keeping only that approach and avoid the redundant config.